modules/ob-enabling-builds-in-a-network-restricted-environment-using-the-buildah-build.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * builds/uninstalling-openshift-builds.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id='ob-creating-buildah-build-in-a-network-restricted-environment_{context}']
```

Comment:
```suggestion
[id='ob-creating-buildah-build-in-a-network-restricted-environment_{context}']
```
`ob-creating-buildah-build-in-a-network-restricted-environment`: this part should be same as the filename (excluding `.adoc`). Please ocnsider renaming this file from `ob-enabling-builds-in-a-network-restricted-environment-using-the-buildah-build.adoc` to something really shorter.